actioncable/lib/action_cable/subscription_adapter/postgresql.rb

```diff
@@ -0,0 +1,99 @@
+gem 'pg', '~> 0.18'
```

Comment:
It's not a bundler definition it's aactivating a specific version of a gem. Most commonly I've seen this done with Minitest to ensure any require uses the gem and not the version shipping with Ruby.